functional_tests.go

```diff
@@ -3327,13 +3842,13 @@ func testComposeObjectErrorCasesWrapper(c *minio.Client) {
 	srcSlice := srcArr[:]
 	dst, err := minio.NewDestinationInfo(bucketName, "object", nil, nil)
 	if err != nil {
-		logger().Fatal(err)
+		failureLog(function, args, startTime, "", "NewDestinationInfo failed", err).Fatal()
 	}
 
 	if err := c.ComposeObject(dst, srcSlice); err == nil {
-		logger().Fatal("Error was expected.")
+		failureLog(function, args, startTime, "", "Expected error in ComposeObject", err).Fatal()
```

Comment:
No point of logging nill error message, err:
failureLog(function, args, startTime, "", "Expected error in ComposeObject"~~, err~~).Fatal()